airflow/values.yaml

```diff
@@ -21,9 +14,10 @@ airflow:
   # set the max number of retries during container initialization
   init_retry_loop:
   # base image for webserver/scheduler/workers
-  image: mumoshu/kube-airflow:1.8.0.0-1.6.1
+  image: puckel/docker-airflow
+  imageTag: 1.9.0
```

Comment:
note: we need to use a more recent version of airflow to support mount in subpath (https://github.com/apache/incubator-airflow/pull/2952). Since airflow 2.0 is not released yet, we are stuck with custom build for this feature. Until then, only root path is supported.

Reply:
Ah, I wasn't aware of custom features in the other docker image build.

Reply:
don't worry, I'll document the limitations.
Also, I think we should, at one point, move to the official helm repository. I'll be glad if you can help me on this :)